hivemind/hivemind_cli/run_server.py

```diff
@@ -18,8 +18,7 @@ def main():
     # fmt:off
     parser = configargparse.ArgParser(default_config_files=["config.yml"])
     parser.add('-c', '--config', required=False, is_config_file=True, help='config file path')
-    parser.add_argument('--listen_on', type=str, default='0.0.0.0:*', required=False,
-                        help="'localhost' for local connections only, '0.0.0.0' for ipv4 '[::]' for ipv6")
+
```

Comment:
Let's add `--host_maddrs` and `--announce_maddrs` arguments forwarded to `hivemind.DHT`/`hivemind.P2P`, like we have in [examples/albert](https://github.com/learning-at-home/hivemind/blob/master/examples/albert/arguments.py#L26).